test/integration/handlers/authorize-handler_test.js

```diff
@@ -332,6 +332,50 @@ describe('AuthorizeHandler integration', function() {
         });
     });
 
+    it('should redirect to an error response if `scope` is insufficient', function() {
```

Comment:
Can you add a test for when `validateScope` isn't included in the model?  Otherwise this PR looks good.